_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
+When you set out to create an alert,
+you'll need to start with a query.
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
+
+Otherwise, if you already have a query and filters,
+you can create a blank alert
+from the top menu (**Alerts & Events > New alert**)
+or from the _Alert definitions_ page
 
 Community plans have a maximum of 50 alerts.
 {:.info-box.note}
 
-If you want help updating an existing alert, you can [skip the first part of this page](#to-configure-an-alert).
-
-#### To start a new alert
+#### To create an alert in Kibana
 
-![Kibana query bar]({{site.baseurl}}/images/kibana/kibana--query-bar.png)
+![Kibana 7 search bar]({{site.baseurl}}/images/kibana/kibana-7-search-bar.png)
 
-##### Set your query in Kibana
+Refine your search query and filters,
+and review the returned logs
+to make sure you got the expected results.
 
-In Kibana, type a query in the query bar
-and press Enter.
-Review the results in the histogram and the document table,
-and make sure your query returned the expected results.
+Alerts support Lucene query language only.
+You won't be able to create an alert
+from a search written in Kibana Query Language (KQL).
+{:.info-box.important}
 
-Click **Create Alert** (to the right of the query bar).
-The _Create Alert Definition_ page is shown.
+Once you're happy with your alert,
+click **Create alert** (above the query bar).
+You're taken to the _New alert_ page.
 
 Continue with [To configure an alert](#to-configure-an-alert).
 
 #### To configure an alert {#to-configure-an-alert}
 
-![Configure an Alert]({{site.baseurl}}/images/alerts/alerts--configure-alert.png)
-
 <div class="tasklist">
 
-##### Name and tag the alert
+##### Name the alert
+
+Give your alert a meaningful name.
+
+If your alert is triggered,
+the name is used as the email subject or Slack heading.
+A helpful name gives the quick need-to-know-now information
+and doesn't use unnecessary words.
 
-Type a **Name** and a detailed **Description**.
+##### Edit the search settings
 
-Add **Tags** to help categorize this alert.
+###### Query and filters
 
-##### _(Optional)_ Edit the search settings
+![Alert query and filters]({{site.baseurl}}/images/alerts/query-and-filters.png)
 
-If you need to, change your alert **Query** and the **Accounts** that the query will search.
+If you need to change your alert query and filters,
+do that now.
 
-If you use an invalid query, the alert will be automatically disabled. Run your query in Kibana so you can be sure you're getting the expected results.
+Your query is an important source of information
+when you're investigating a triggered alert.
+Shorten your query by using filters whenever you can.
+
+If you use an invalid query,
+the alert will be automatically disabled.
+To make sure it's valid,
+test your query in Kibana.
```

Comment:
We have the ability in the alerts page to say that the query is invalid, and the user won't be able to save the alert.
@imnotashrimp I don't think we've gone over the error message if the invalid syntax.

Reply:
Wrote a string for the error message just now, sent privately. 

@AmirTugi So I can remove this bit completely then? Users are prevented from saving an invalid query?

Reply:
Removed